setup.py

```diff
@@ -1,7 +1,10 @@
 from setuptools import setup, find_packages
 import os
 
-version = '1.1.dev1'
+name = 'collective.opengraph'
+path = ['src'] + name.split('.') + ['version.txt']
+version = open(os.path.join(*path)).read().strip()
+
```

Comment:
Is this change with keeping compatibility with tools like zest.releaser to make releases?